### ISSUE TITLE
cannot install nginx example

### ISSUE THREAD

**Issue by michelleN** [kubernetes/helm#879]
```
helm [689-template-rules]$ bin/helm lint docs/examples/nginx/
[ERROR] templates: "templates/_helpers.tpl" needs to use the .yaml extension
helm [689-template-rules]$ bin/helm install docs/examples/nginx/
Error: release auxilliary-rattlesnake failed: [Service "auxilliary-rattlesnake-nginx" is invalid: metadata.name: Invalid value: "auxilliary-rattlesnake-nginx": must be a DNS 952 label (at most 24 characters, matching regex [a-z]([-a-z0-9]*[a-z0-9])?): e.g. "my-name", yaml: control characters are not allowed]
Usage:
  helm install [CHART] [flags]

Flags:
      --dry-run         simulate an install
  -n, --name string     the release name. If unspecified, it will autogenerate one for you.
  -f, --values string   path to a values YAML file

Global Flags:
      --debug              enable verbose output
      --home string        location of your Helm config. Overrides $HELM_HOME. (default "$HOME/.helm")
      --host string        address of tiller. Overrides $HELM_HOST.
      --namespace string   kubernetes namespace
```

I realized the above example was on a branch so this error is from master:
```
helm [master]$ bin/helm install docs/examples/nginx/
Error: release cold-vulture failed: yaml: control characters are not allowed
Usage:
  helm install [CHART] [flags]

Flags:
      --dry-run         simulate an install
  -n, --name string     the release name. If unspecified, it will autogenerate one for you.
  -f, --values string   path to a values YAML file

Global Flags:
      --debug              enable verbose output
      --home string        location of your Helm config. Overrides $HELM_HOME. (default "$HOME/.helm")
      --host string        address of tiller. Overrides $HELM_HOST.
      --namespace string   kubernetes namespace
```

**Comment by technosophos** [comment]
The bug is in the template. I'll fix it.

**Comment by technosophos** [comment]
Fixed by #883
